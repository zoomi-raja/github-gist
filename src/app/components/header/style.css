.header{
  display: flex;
  justify-content: center;
}
.header__search{
  width: 50%;
  padding: 1.2rem;
}