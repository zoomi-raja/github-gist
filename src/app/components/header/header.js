import { memo } from "react";
import PropTypes from 'prop-types';

import './style.css';
function header({search,setSearch}) {    
  return (
    <div className='header'>
      <input
       type='search'
       className='header__search' 
       placeholder='Search By Name..!' 
       value={search} 
       onChange={setSearch}/>
    </div>
  )
}
header.propTypes = {
  search: PropTypes.string,
  setSearch: PropTypes.func
};
export const Header = memo(header);