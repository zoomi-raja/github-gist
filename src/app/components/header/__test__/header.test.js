import renderer from 'react-test-renderer';
import { Header } from '../header';
it('header snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
})