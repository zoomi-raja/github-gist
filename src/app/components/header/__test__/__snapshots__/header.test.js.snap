// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header snapshot 1`] = `
<div
  className="header"
>
  <input
    className="header__search"
    placeholder="Search By Name..!"
    type="search"
  />
</div>
`;
